src/ZoneServer/World/Maps/Map.cs

```diff
@@ -190,8 +191,20 @@ private void Disappearances()
 			lock (_monsters)
 				toDisappear = _monsters.Values.Where(a => a.DisappearTime < now).ToList();
 
-			foreach (var monster in toDisappear)
+			foreach (var monster in toDisappear) {
```

Comment:
Please try to conform to the general code style, as well as the coding conventions layed out in [CONTRIBUTING.md](https://github.com/NoCode-NoLife/melia/blob/master/CONTRIBUTING.md). Curly braces should be placed on their own line.

Reply:
Looks like I missed those two.  It's been fixed.